src/createServer.js

```diff
@@ -1,3 +1,68 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url || '', `http://${req.headers.host}`);
+
+    const userText = url.pathname.slice(1);
+    const typeOfCase = url.searchParams.get('toCase');
+    const errors = [];
+    const typesOfCase = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    if (!userText) {
+      errors.push({
+        message:
+          'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
Create some object which will store messages with specific keys and reuse it here.